asapdiscovery-docking/asapdiscovery/docking/workflows/large_scale_docking.py

```diff
@@ -51,10 +52,12 @@ class LargeScaleDockingInputs(BaseModel):
         Whether to upload the results to Postera.
     postera_molset_name : str, optional
         The name of the molecule set to pull from and/or upload to.
-    du_cache : str, optional
-        Path to a directory where design units are cached
-    gen_du_cache : str, optional
-        Path to a directory where generated design units should be cached
+    cache_dir : str, optional
+        Path to a directory where structures are cached
+    gen_cache : str, optional
+        Path to a directory where prepped structures should be cached
```

Comment:
I think it's still not clear to me what the distinction between these two is. can you either unify them or make their descriptions a bit more detailed? also, less important but it seems like the description here doesn't match the `Field` description below